apps/web-client/src/lib/stores/inventory/__tests__/table_content.test.ts

```diff
@@ -24,4 +29,187 @@ describe('tableContentStore', () => {
 			totalItems: 0
 		} as PaginationData);
 	});
+
+	test('should create display store from db and entries streams', async () => {
+		const db = await newTestDB();
+
+		await db.upsertBook({
+			isbn: '0195399706',
+			title: 'The Age of Wonder',
+			authors: 'Richard Holmes',
+			publisher: 'HarperCollins UK',
+			year: '2008',
+			price: 69.99
+		});
+
```

Comment:
LGTM, can we just:
- rename this to: "should stream book data with the entires (matching their isbn)"
- reduce the number of books to two (it's a lot of code, and we don't really need three entries)
- save `book1` and `book2` as variables and expect `{ ...entry, ...book }` (to reduce the code further)
- also use `Promise.all()` for setup (slightly quicker tests)